neos/neos-development-collection
# Bugfix/breadcrumbmenu escapes special characters

**What I did**

Prevent escaping of special characters in breadcrumb ite lables.

**How I did it**

Piped output for the label through the f:format.raw() ViewHelper.

**How to verify it**

* Create  page with special characters in the title
* Use the Prototype **TYPO3.Neos:BreadcrumbMenu** in your layout
* View the page in the frontend, the label for the current page should not contain special chars anymore

**Checklist**
